source/mainnet/net/guides/run-node.rst

```diff
@@ -378,28 +419,26 @@ To retrieve mainnet node logs run:
          - CONCORDIUM_NODE_CONNECTION_BOOTSTRAPPING_INTERVAL=1800
          # Haskell RTS flags to pass to consensus. `-N2` means to use two threads
          # for consensus operations. `-I0` disables the idle garbage collector
-         # which reduces CPU load for non-baking nodes.
+         # which reduces CPU load for passive nodes.
          - CONCORDIUM_NODE_RUNTIME_HASKELL_RTS_FLAGS=-N2,-I0
+
        entrypoint: ["/concordium-node"]
+
        # Exposed ports. The ports the node listens on inside the container (defined
        # by `CONCORDIUM_NODE_LISTEN_PORT` and `CONCORDIUM_NODE_RPC_SERVER_PORT`)
        # should match what is defined here. When running multiple nodes the
        # external ports should be changed so as not to conflict.
-       # In the mapping below, the first port is the `host` port, and the second
-       # port is the `container` port. When the `container` port is changed the
-       # relevant environment variable listed above must be changed as well. For
-       # example, changing `10000:10000` to `10000:13000` would mean that
-       # `CONCORDIUM_NODE_RPC_SERVER_PORT` should be set to `13000`. Otherwise
-       # the node's gRPC interface will not be available from the host.
        ports:
        - "8888:8888"
        - "10000:10000"
        - "20000:20000"
+
        volumes:
        # The node's database should be stored in a persistent volume so that it
        # survives container restart. In this case we map the **host** directory
-       # /var/lib/concordium-mainnet to be used as the node's database directory.
-       - /var/lib/concordium-mainnet:/mnt/data
+       # ~/tmp/mainnet to be used as the node's database directory.
+       - /tmp/mainnet:/mnt/data
```

Comment:
I don't think this change should have been made. It is not related to any code changes we have done, and what was there was fine.